x-pack/plugins/infra/public/components/waffle/waffle_accounts_controls.tsx

```diff
@@ -24,7 +24,7 @@ interface Props {
 
 export const WaffleAccountsControls = (props: Props) => {
   const { accountId, options } = props;
-  const [isOpen, setIsOpen] = useState();
+  const [isOpen, setIsOpen] = useState<boolean>();
```

Comment:
To understand, are these added explicits types are a requirement for the bump or only cleanup?

Reply:
> To understand, are these added explicits types are a requirement for the bump

yeah, it started showing error after TS version upgrade